src/demo/colors/ColorPanel.tsx

```diff
@@ -39,18 +38,20 @@ export function ColorPanel(props: ColorPanelProps) {
             const copyText = `var(${varName})`;
             return (
                 <div className="color-panel__card" key={color.name}>
-                    <ReactCopyToClipboard text={copyText}>
-                        <div
-                            className={`color-panel__card-box ${boxBorders}`}
-                            style={{background: `var(${varName})`}}
-                        />
-                    </ReactCopyToClipboard>
+                    <CopyToClipboard text={copyText}>
+                        {() => (
```

Comment:
Let's support normal children type as well as it was in `ReactCopyToClipboard `? I find it usefull, if I don't need status feedback